board/miovision/smartsense/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020, Miovision.
+# Copyright (c) 2020, Miovision
```

Comment:
Did you want to have the copyright notice twice?

Reply:
Fixed